webrender/src/tiling.rs

```diff
@@ -960,11 +959,13 @@ pub struct RenderPass {
 }
 
 impl RenderPass {
-    fn new(pass_index: isize, is_framebuffer: bool) -> RenderPass {
+    fn new(pass_index: isize,
+           is_framebuffer: bool,
+           size: DeviceUintSize) -> RenderPass {
```

Comment:
Fixed - had to change the code structure slightly to make this work, but it's much cleaner now.